.scrutinizer.yml

```diff
@@ -29,6 +29,4 @@ build:
                 - php-scrutinizer-run
     tests:
         override:
-        - true
-
-
+        - true # Disable unit tests because they are handled by Travis
```

Comment:
set to `false` :question:

Reply:
Oddly enough, setting "true" will skip this according to the docs.